tagController.js

```diff
@@ -123,9 +165,15 @@ function tagList(tagRowId, tagRowSelect){
         tagOption.value = tagItem.id;
         tagOption.text = tagItem.name;
 
-        if(mainUnitData.tagIds.includes(tagItem.id.toString())){
-            tagOption.disabled = 'true';
+        //fix me
+        let unitTagsCounter;
+        for(unitTagsCounter = 0; unitTagsCounter < mainUnitData.tags.length; unitTagsCounter++){
+            let tagData = mainUnitData.tags[unitTagsCounter];
+            if(tagData.id == tagItem.id){
```

Comment:
In JS, use `===` for comparisons (I know, I forget sometimes too)

Reply:
in some cases, its pulling the html value of a number.